content/assets/blog/automatic-classifications-for-peace-invite/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Automatic Classifications for Peace Invite
+date: "2023-04-14T04:23:05Z"
+description: "Are you interested in joining the Automatic Classifications for Peace group?"
+author: benjamin
+---
+Hello!,
+
+datavaluepeople (https://datavaluepeople.com/) and Build Up (https://howtobuildup.org/) are working on a new project, the Automatic Classifications for Peace group, and we would love to know if you might be interested in joining.
+
+The Automatic Classifications for Peace group helps to further the development of open-source text classification systems for peacebuilding and mediation. Build Up has been developing classification systems for peacebuilding in collaboration with datavaluepeople: with a set of open-source social media analysis tools collectively called Phoenix: https://gitlab.com/howtobuildup/phoenix. Phoenix is also a process that guides peacebuilders and mediators in conducting relevant social media analysis. The Phoenix process and tools have already been used by 100s of local peacebuilders and helped them find valuable insight into ways they can help the communities they care about. 
+
+A key component of Phoenix is the capacity to build customizable automatic classification models that support the analysis needs of peacebuilders and mediators. We know other organisations are also interested in this particular capability. With Automatic Classifications for Peace, we hope to widen and deepen the number of organisations involved in open-source text classification systems for peacebuilding. The project will build the structures necessary so that we can collaborate to share collective knowledge and build things on top of each other rather than siloed. Our aim is to create tools and code repositories that are accessible, easy to use and can attract other technologists to work on them too, as well as to release software tools, guidance and documentation directed towards peacebuilders with different levels of tech literacy.
+
+Ultimately, we hope to make tools reusable by any local peacebuilder that can do automatic text classification of social media discourse in the communities and languages peacebuilders think are important. These tools will be connected to the community of peacebuilders that is already using Phoenix - a community which Build Up is fronting and helps peacebuilders that are looking at how they can use social media analysis.
+
+The Automatic Classifications for Peace group will be meeting online a number of times throughout the year, leading up to a real-life 2-day workshop in Nairobi on Nov 29 & 30, pre-Build Up’s yearly international peacebuilding conference Build Peace 2023. These meetings will be facilitated by datavaluepeople and Build Up and will hopefully create the relationships and structure for the group to be fun and worthwhile.
```

Comment:
"pre-Build Up’s" -> "just before Build Up's ... in the same location."

Reply:
"the group to be fun and worthwhile." -> "the group to be effective, worthwhile, and fun."